### PR TITLE
Organizing getting started redirects and adding search terms

### PR DESCRIPTION
**Pull request by runleonarun**
## Description & motivation
* Added a redirect for create-a-project which I deleted becuse it was an old landign page that didn't make sense in the new architecture.
* Added keywords to improve search for "create a project" and related terms

## Checklist
If you removed existing pages (delete if not applicable):
- [x] The page has been removed from `website/sidebars.js`
Note: this page had already been deleted form tne sidebar
- [x] An entry has been added to `_redirects`

### CODE REVIEW

File: _redirects [hunk 1]
```diff
@@ -1,16 +1,36 @@
+# add new redirects to the top because they will override later ones
+# getting started guide
+/guides/getting-started /docs/get-started/getting-started/overview 301
```
Comment (runleonarun) [inline _redirects:3]:
```suggestion
/guides/getting-started /docs/get-started/getting-started/overview 301
/docs/get-started/getting-started/create-a-project /docs/get-started/getting-started/build-your-first-models 301
```

File: website/docs/docs/get-started/getting-started/building-your-first-project/build-your-first-models.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 title: Build your first models
 id: build-your-first-models
 description: "Now that you've set up the starter project, you can get to the fun part — building models!"
+keywords: [projects, create a project, dbt project]
 ---
 
 Now that you set up your sample project and had a successful run, you can get to the fun part — [building models](/docs/build/sql-models)! You will take a sample query and turn it into a model in your dbt project. A model in dbt is a select statement.
```
Comment (runleonarun) [inline website/docs/docs/get-started/getting-started/building-your-first-project/build-your-first-models.md]:
```suggestion
After setting up a sample project with a successful run, you can get to the fun part — [building models](/docs/build/sql-models)! You will take a sample query and turn it into a model in your dbt project. A model in dbt is a select statement.
```

File: _redirects [hunk 3]
```diff
@@ -1,16 +1,37 @@
...
+/docs/get-started/getting-started/create-a-project /docs/get-started/getting-started/build-your-first-models 301
```
Comment (runleonarun) [inline _redirects]:
```suggestion
/docs/get-started/getting-started/create-a-project  /docs/get-started/getting-started/building-your-first-project/build-your-first-models 301
```

File: _redirects [hunk 4]
```diff
@@ -1,16 +1,37 @@
...
+/guides/getting-started/building-your-first-project /docs/get-started/getting-started/building-your-first-project/build-your-first-models 301
+/guides/getting-started/building-your-first-project/build-your-first-models /docs/get-started/getting-started/building-your-first-project/build-your-first-models 301
+/guides/getting-started/building-your-first-project/schedule-a-job /docs/get-started/getting-started/building-your-first-project/schedule-a-job 301
+/guides/getting-started/building-your-first-project/test-and-document-your-project /docs/get-started/getting-started/building-your-first-project/test-and-document-your-project 301
+/guides/getting-started/create-a-project /docs/get-started/getting-started/build-your-first-models 301
```
Comment (runleonarun) [inline _redirects]:
```suggestion
/guides/getting-started/create-a-project /docs/get-started/getting-started/building-your-first-project/build-your-first-models301
```

File: website/docs/docs/get-started/getting-started/building-your-first-project/build-your-first-models.md [hunk 5]
```diff
@@ -2,9 +2,10 @@
 title: Build your first models
 id: build-your-first-models
 description: "Now that you've set up the starter project, you can get to the fun part — building models!"
+keywords: [projects, create a project, dbt project, get started with Cloud]
 ---
 
-Now that you set up your sample project and had a successful run, you can get to the fun part — [building models](/docs/build/sql-models)! You will take a sample query and turn it into a model in your dbt project. A model in dbt is a select statement.
+After setting up a sample project with a successful run, you can get to the fun part — [building models](/docs/build/sql-models)! You will take a sample query and turn it into a model in your dbt project. A model in dbt is a select statement.
```
Comment (runleonarun) [inline website/docs/docs/get-started/getting-started/building-your-first-project/build-your-first-models.md]:
```suggestion
After setting up a sample project with a successful run, you can get to the fun part — [building models](/docs/build/sql-models)! You will learn how to take a sample query and turn it into a model in your dbt project. A model in dbt is a select statement.
```

**Review by nghi-ly — DISMISSED** [review]
🚢

File: _redirects [hunk 6]
```diff
@@ -1,16 +1,37 @@
+# add new redirects to the top because they will override later ones
+# getting started guide
+/guides/getting-started /docs/get-started/getting-started/overview 301
+/docs/get-started/getting-started/create-a-project  /docs/get-started/getting-started/building-your-first-project/build-your-first-models 301
```
Comment (nghi-ly) [inline _redirects]: sugg: 

the project creation content is at https://docs.getdbt.com/docs/get-started/getting-started/getting-set-up/setting-up-bigquery#initialize-your-dbt-project. it's not good practice to redirect to an anchor link so maybe consider "About set up"  https://docs.getdbt.com/docs/get-started/getting-started/set-up-dbt-cloud

Comment (runleonarun) [inline _redirects]:
```suggestion
/docs/get-started/getting-started/create-a-project  /docs/get-started/getting-started/set-up-dbt-cloud 301
```

File: website/docs/reference/warehouse-setups/dremio-setup.md [hunk 7]
```diff
@@ -88,10 +88,6 @@ Next, configure the profile for your project.
 
 When you initialize a project, you create one of these three profiles. You must configure it before trying to connect to Dremio Cloud or Dremio Software.
 
-## Profiles
-
-When you initialize a project, you create one of these three profiles. You must configure it before trying to connect to Dremio Cloud or Dremio Software.
-
 * Profile for Dremio Cloud
```
Comment (runleonarun) [inline website/docs/reference/warehouse-setups/dremio-setup.md]:
```suggestion
## Profiles

When you initialize a project, you create one of these three profiles. You must configure it before trying to connect to Dremio Cloud or Dremio Software.

* Profile for Dremio Cloud
```

File: website/docs/reference/warehouse-setups/dremio-setup.md [hunk 8]
```diff
@@ -177,7 +177,7 @@ For descriptions of the configurations in these profiles, see [Configurations](#
 ### Configurations in Profiles for Dremio Cloud
 | Configuration | Required? | Default Value | Description |
 | --- | --- | --- | --- |
-| `cloud_host` | Yes | `https://api.dremio.cloud` | US Control Plane: `https://api.dremio.cloud`<br></br>EU Control Plane: `https://api.eu.dremio.cloud` |
+| `cloud_host` | Yes | `https://api.dremio.cloud` | US Control Plane: `https://api.dremio.cloud`<br/>EU Control Plane: `https://api.eu.dremio.cloud` |
```
Comment (runleonarun) [inline website/docs/reference/warehouse-setups/dremio-setup.md]:
```suggestion
| `cloud_host` | Yes | `https://api.dremio.cloud` | US Control Plane: `https://api.dremio.cloud`<br></br>EU Control Plane: `https://api.eu.dremio.cloud` |
```

**Pull request merged**